recipes/runit_systemd.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright:: Copyright (c) 2015 Opscode, Inc.
+#
+# All Rights Reserved
+#
+
+project_name = node['enterprise']['name'].clone
+project_name.gsub!(/_/, '-') if project_name == 'private_chef'
+unit_name = "#{project_name}-runsvdir-start.service"
+
+template "/usr/lib/systemd/system/#{unit_name}" do
+  owner "root"
+  group "root"
+  mode "0644"
+  variables({
+              :install_path => node[node['enterprise']['name']]['install_path'],
+              :project_name => project_name
+  })
+  source "runsvdir-start.service.erb"
+end
+
+execute "systemctl enable #{unit_name}" do
+  not_if "systemctl is-enabled #{unit_name}"
+end
+
+execute "systemctl start #{unit_name}" do
+  not_if "systemctl is-active #{unit_name}"
+end
```

Comment:
sysvinit can't, its directly editing /etc/inittab and HUP'ing init, there's no provider for that.   i have no idea why the upstart job wound up using initctl instead of start/stop in the service provider, there might be history there or it might just be cargo culted from bad code years ago.  we shouldn't continue a bad pattern if we don't have to though.